public/cached.php

```diff
@@ -22,7 +22,7 @@
 
 $dbh = include('../lib/db.php');
 // saves the report to the database
-$sth = $dbh->prepare(" SELECT * FROM {$db_reports_table} WHERE course_id = :courseid ORDER BY date_run DESC");
+$sth = $dbh->prepare(" SELECT id, date_run, errors, suggestions FROM {$db_reports_table} WHERE course_id = :courseid ORDER BY date_run DESC");
```

Comment:
The columns were specified because this table now contains a huge report that's not needed for the view.